### DISCUSSION TITLE
ISO of CasaOS

### DISCUSSION THREAD

**Discussion by zarevskaya** [IceWhaleTech/CasaOS#209] [Ideas]
Hi,

It would be good to have the ISO of CasaOS to reinstall the ZIMABOARD.

:0)

Have a nice day, Stay safe!

**Comment by M1n-74316D65** [comment]
Found it.
[https://docs.zimaboard.com/faq/Restore-factory-settings.html](https://docs.zimaboard.com/faq/Restore-factory-settings.html)
